Library/Homebrew/cmd/install.rb

```diff
@@ -142,6 +142,8 @@ def install
     rescue FormulaUnavailableError => e
       if (blacklist = blacklisted?(e.name))
         ofail "#{e.message}\n#{blacklist}"
+      elsif e.name == "updog"
+        ofail "What's updog?"
```

Comment:
Just want to ask whether it's better to put this Easter egg into `blacklist` rather than a separate if else branch.